fashion-messaging-app-api/server.js

```diff
@@ -21,7 +21,7 @@ const app = express();
 const API_KEY = "06ZcwjgbmJBM8T2TxLUZ5iwdXXGxiAgz0Z018b7QPKwR1ExipkFjaAuw";
 const clientAPI = createClient(API_KEY);
 const defaultQuery = "Fashion";
-const PHOTOS = 8;
+const PHOTOS = 2;
```

Comment:
This is related to the rate limit and caching that I explained yesterday. Since my cache is lost each time I run my server, I don't want to make too many calls and utilize the entire rate limit. I would increase number of photos once I am almost done with the testing stage when I know I won't be restarting my server too frequently